tests/test_kernel32/test_handle.py

```diff
@@ -120,3 +122,86 @@ def test_process_dies_before_timeout(self):
         self.assertEqual(
             WaitForSingleObject(hProcess, library.INFINITE),
             library.WAIT_OBJECT_0)
+
+
+class TestGetHandleInformation(TestCase):
+    """
+    Tests for :func:`pywincffi.kernel32.GetHandleInformation`
+    """
+    def test_get_handle_info_stdin(self):
+        _, library = dist.load()
+        stdin_handle = GetStdHandle(library.STD_INPUT_HANDLE)
+        handle_flags = GetHandleInformation(stdin_handle)
+        inherit = handle_flags & library.HANDLE_FLAG_INHERIT
+        expected = (0, 1)
```

Comment:
I'd modify this a bit so we're testing specific values depending on the version of Python.  That will make it clear what the expectation is per Python version.